test/log_helper.erl

```diff
@@ -0,0 +1,20 @@
+%% Helper module for checking expected log messages
+%% For assertions, see log_helper.hrl
+
+-module(log_helper).
+-compile([export_all, nowarn_export_all]).
+
+%% Call in init_per_testcase
+set_up() ->
```

Comment:
just note, if doing it in parallel, it would stop working because see:

https://github.com/arcusfelis/logger_debug_h/blob/main/src/logger_debug_h.erl#L23

currently it is sequential in this case.

Reply:
I did indeed manage to get a fail after 1000 retries of many parallel tests. I will investigate the reason, if I have time.
Sequential is good enough for now, but we could investigate if it's a bug.

Reply:
Just FYI: ETS-based solution with a single handler is still very short, and works as expected.